bower.json

```diff
@@ -29,6 +29,7 @@
     "core-doc-viewer": "Polymer/core-doc-viewer#0.8-preview",
     "core-icon": "Polymer/core-icon#0.8-preview",
     "core-icons": "Polymer/core-icons#0.8-preview",
+    "test-fixture": "Polymer/test-fixture#master",
```

Comment:
Yeah, that will create an issue where bower will prompt you to pick one of the two. What do you think we should do about that?